src/main/kotlin/org/rust/ide/inspections/fixes/AttachFileToModuleFix.kt

```diff
@@ -193,3 +201,5 @@ fun withMockModuleAttachSelector(
         MOCK = null
     }
 }
+
+private val RsMod.firstItem: RsElement? get() = itemsAndMacros.firstOrNull { it !is RsAttr && it !is RsVis }
```

Comment:
Do I understand correctly, that `RsMod` is always `RsFile` in this quick fix?
If yes then `it !is RsVis` looks redundant because `RsFile` cannot contain visibility element as a direct child. In auto-import implementation, it makes sense since we can insert `use` item into inline module

Reply:
Also, if we use only `RsFile`s, it would be great to use them instead of `RsMod` to provide more info about the context to the reader

Reply:
`RsMod` is indeed always `RsFile`, I changed it to be more concrete, thanks.